recipes/deepchopper/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "deepchopper" %}
+{% set version = "1.2.4" %}
+{% set sha256 = "e7f0bfed8c34e011e924bc12e30dec9c58777f1879962b9a2036da29009461e4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py < 310]
+  run_exports:
+    - {{ pin_subpackage("deepchopper", max_pin="x") }}
+  entry_points:
+    - deepchopper = deepchopper.cli:app
+    
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}
+    - make
+    - cmake
+  host:
+    - python
+    - pip
+    - maturin >=1.2.1,<2
+    - setuptools-rust
+    - setuptools
+  run:
+    - python
+    - torch >=2.1.0
```

Comment:
The ARM64 builds fail due to:
```
11:23:33 BIOCONDA INFO (OUT) Encountered problems while solving:
11:23:33 BIOCONDA INFO (OUT)   - nothing provides requested torch >=2.1.0
```
But https://anaconda.org/search?q=torch does not list such package in `conda-forge` and/or `bioconda` channels at all.
I guess the same error will be reported for x86_64 once the CI issue is solved.
Did you mean https://anaconda.org/conda-forge/pytorch ?

Reply:
@martin-g, thank you for your excellent help! I just realize I made a mistake. The PyTorch package names are different for conda and pip:
In conda:  'pytorch'
In pip: 'torch'"

I've found two good PyTorch options:

The conda-forge version (conda-forge/pytorch) which works great for my needs
The official PyTorch channel (pytorch/pytorch) which provides the latest releases

Both are available through Anaconda, and I can work with either option.
Again, Thanks for your help!